pkg/apis/stork/v1alpha1/action.go

```diff
@@ -34,27 +34,100 @@ type Action struct {
 
 // ActionSpec specifies the type of Action
 type ActionSpec struct {
-	ActionType ActionType `json:"actionType"`
+	ActionType      ActionType       `json:"actionType"`
+	ActionParameter *ActionParameter `json:"actionParameter"`
 }
 
 // ActionType lists the various actions that can be performed
 type ActionType string
 
 const (
 	// to start apps on destination cluster
+	ActionTypeNearSyncFailover = "nearsyncFailover"
```

Comment:
Shouldn't it be  "ActionType NearSyncFailover"

Reply:
I guess it should be `ActionTypeNearSyncFailover ActionType = "nearsyncFailover"`

Reply:
Done